<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clinet</class>
 <widget class="QMainWindow" name="Clinet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clinet</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Buttom_Fitness">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/picture/fitness.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Diet">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/picture/diet.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_chatroom">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/picture/chatroom.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_fitness">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>fitness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_diet">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>diet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_chatroom">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>chatroom</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TB_chatroom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_send">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_send">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_back">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_connect">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>TB_chatroom</zorder>
   <zorder>Buttom_Fitness</zorder>
   <zorder>Button_Diet</zorder>
   <zorder>Button_chatroom</zorder>
   <zorder>label_fitness</zorder>
   <zorder>label_diet</zorder>
   <zorder>label_chatroom</zorder>
   <zorder>LE_send</zorder>
   <zorder>Button_send</zorder>
   <zorder>Button_back</zorder>
   <zorder>Button_connect</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
